package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "appengine-cloudtasks",
+  "description": "Google App Engine Cloud Tasks example.",
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
```

Comment:
I don't think this is right. We could just remove author, license, descp etc as we don;t plan to publish this

Reply:
True, I removed those fields.